src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,96 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+import { evaluateOnce } from '../derived/evaluateOnce';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
+  let returnMethod;
+  const newValue = new Value(0);
+  const newClock = new Clock();
+  const currentState = AnimationClass.getDefaultState();
+  currentState.position = newValue;
+  let alwaysNode;
+  let isStarted = false;
+  let isDone = false;
+  let wasStopped = false;
+  return {
+    start: currentReturnMethod => {
+      if (isStarted) {
+        returnMethod && returnMethod({ finished: false });
+        return;
+      }
+      if (isDone) {
+        console.warn('Animation has been finished before');
+        // inconsistent with React Native
+        return;
+      }
+      isStarted = true;
+      evaluateOnce(
+        set(currentState.position, value),
+        currentState.position,
+        () => {
+          alwaysNode = always(
+            set(
+              value,
+              block([
+                cond(clockRunning(newClock), 0, startClock(newClock)),
+                node(newClock, currentState, config),
+                cond(currentState.finished, [
+                  call([], () => {
+                    isStarted = false;
+                    isDone = true;
+                    value.__setAnimation(null, !wasStopped);
+                    if (!wasStopped) {
+                      wasStopped = false;
+                    }
+                  }),
+                  stopClock(newClock),
+                ]),
+                currentState.position,
+              ])
+            )
+          );
+          returnMethod = currentReturnMethod;
```

Comment:
What's the point of having `returnMethod` and `currentReturnMethod`? Looks like if you rename `currentReturnMethod` -> `returnMethod` it would have the same effect

Reply:
I wish to have access to method out of the scope, but now it's not needed